Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyDoubleSpinCtrl.cpp

```diff
@@ -85,9 +85,9 @@ namespace AzToolsFramework
         m_pSpinBox->setValue(value);
         m_pSpinBox->blockSignals(false);
 
-        //the value didn't change, so don't send a notify
-        if (!AZ::ClampIfCloseMag(value, oldValue) && notifyLater)
```

Comment:
Do you know why a value is evaluated as a condition here?

Reply:
I couldn't figure out the purpose of `!AZ::ClampIfCloseMag(value, oldValue)`.

ClampIfCloseMag returns either x or y so it didn't make sense as a conditional.
`//! ClampIfCloseMag(x, y, epsilon) returns y when x and y are within epsilon of each other`

Reply:
My hunch is it should be, but maybe we can keep it unchanged in this PR?

```cpp
if (!AZ::IsCloseMag(value, oldValue) && notifyLater)
{
    ...
}
```